guides/build_integration.md

```diff
@@ -0,0 +1,61 @@
+# Build an Integration for MeiliSearch
+
+Here are some guidelines that you should follow, as we want to guarantee consistency and coherence across the different integrations, as well as technical quality for the different MeiliSearch users.
+
+## 1. Main Principle: User Experience First
+
+The provided package should take into account the user experience: it should be easy to use by the user and as intuitive as possible. The customization is important, but the basic usage of the plugin should be accessible.
+
+Feel free to [contact us](https://docs.meilisearch.com/learn/what_is_meilisearch/contact.html#contact-us) if you hesitate about an implementation choice.
+
+## 2. README
+
+Here are the minimum structure that the README must contain:
+
+- A well-explained `Getting Started` section: the user should be able to copy/paste the code and command examples and everything should work as expected.
+- Basic code examples for the most common actions.
+
+We recommend checking out the READMEs of our other integration repositories.
+
+## 3. CONTRIBUTING.md
```

Comment:
FYI

For now it focus on code contribution, we'll suggest updates soon for the template and all repos.